Flush all open files after showing files
By Yauheni Kaliuta <yauheni.kaliuta@nokia.com>

Ported to BusyBox 1.19 by Dennis Groenen <tj.groenen@gmail.com> - 2011-08-19
---

--- a/coreutils/ls.c
+++ b/coreutils/ls.c
@@ -701,6 +701,7 @@ static void display_files(struct dnode *
 			}
 		}
 		putchar('\n');
+		fflush(NULL);
 		column = 0;
 	}
 }
